src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -84,6 +93,74 @@ ACA_OVS_L2_Programmer &ACA_OVS_L2_Programmer::get_instance()
   return instance;
 }
 
+bool ACA_OVS_L2_Programmer::is_ip_on_the_same_host(const std::string host_ip)
+{
+  return std::find(this->host_ips_vector.begin(), this->host_ips_vector.end(),
+                   host_ip) != this->host_ips_vector.end();
+}
+
+void ACA_OVS_L2_Programmer::get_local_host_ips()
```

Comment:
If this method comes from a third-party lib, the best practice is to acknowledge and put the license statement of third party lib in our license statement in this file.